src/functions/updateTechRecordStatus.ts

```diff
@@ -0,0 +1,28 @@
+import TechRecordsDAO from "../models/TechRecordsDAO";
+import TechRecordsService from "../services/TechRecordsService";
+import HTTPResponse from "../models/HTTPResponse";
+import {APIGatewayEvent} from "aws-lambda";
+import {STATUS, HTTPRESPONSE} from "../assets/Enums";
+import S3BucketService from "../services/S3BucketService";
+import { S3 } from "aws-sdk";
+
+export async function updateTechRecordStatus(event: APIGatewayEvent) {
+    const s3BucketService = new S3BucketService(new S3());
+    const techRecordsService = new TechRecordsService(new TechRecordsDAO(), s3BucketService);
+
+    const vin = event.pathParameters!.vin;
```

Comment:
type for vin? Should always be a string

Reply:
You're right, will add it. Strange that tslint didn't complain

Reply:
Done